webapp/src/app/components/load-map/load-map.component.scss

```diff
@@ -0,0 +1,38 @@
+@import "~@angular/material/theming";
+
+$config: mat-typography-config();
+
+
+.mapTree ul,
+.mapTree li {
+	margin-top: 0;
+	margin-bottom: 0;
+	padding-left: 16px;
+	list-style-type: none;
+}
+
+.mapTree li > .mat-button {
+	text-transform: none;
+	width: 100%;
+	min-height: 48px;
+}
+
+.mapTree li > .mat-button /deep/ .mat-button-wrapper {
```

Comment:
Removing this would center the text and icon of every button which is not desirable.
![image](https://user-images.githubusercontent.com/2082553/62199382-ad3c2b00-b383-11e9-839e-4729b8274a8d.png)

Reply:
adding a class to the button with `text-align: left;` fixes it